src/detail_print.js

```diff
@@ -18,23 +19,14 @@ async function waitForElement(selector) {
   return element;
 }
 
-/**
- * Returns a value indicating whether the payment is complete.
- * @return {boolean} Whether payment is completed
- */
-function isPaymentCompleted() {
-  const payStatus = document.querySelector('tr[ng-if*="report.statusString"] > td').textContent;
-  return payStatus === 'Completed';
-}
-
 /**
  * Get the details for a reg data table entry.
  * @param {string} fieldName - The text field name from the reg details table
  * @return {string} The value of the table
  */
 function getDetail(fieldName) {
-  const regDetailTable = document.querySelector('div[ng-include*="data.html"] > table');
-  const th = [...regDetailTable.querySelectorAll('th')]
+  // const regDetailTable = document.querySelector('div[ng-include*="data.html"] > table');
```

Comment:
Nitpick, this will be stored in git history if we ever need it. Should just delete the whole line  rather than leaving commented out code laying around.

Reply:
Left in by mistake while testing, removed!